### DISCUSSION TITLE
I would like to have 2 distinc entities for the average electricity price.

### DISCUSSION THREAD

**Discussion by mintgroen** [JaccoR/hass-entso-e#205] [Ideas]
One with the VAT+profit etc and one just the bare market price.

The reason is that I want to be able to feed the total cost to a display for awareness of other household members.

Next to displaying, I want the bare market price to be able to feed into the energy management system to control the solar production and plan battery charging/discharging.

I can probbly get it to work in NodeRed and with some YAMLing in HA too. I thgin kthat more people would find this useful and nicely integrated into one spot.

**Comment by ThorAlex87** [comment]
You can just make multiple instances of the integration with different price modifier templates, I run five and display all of them in a apexcharts graph.

**Reply by Roeland54** [reply]
This is also al possibility. You will just fetch the data from entso-e for each instances. Which is less efficient but not a real issue.

**Comment by Roeland54** [comment]
You could just get the bare market price from the integration and make as much template helper entities as you need for derived values of the bare price.
